inscriptions/ISic3073.xml

```diff
@@ -179,18 +184,35 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#VM">
+                <div type="textpart" n="A"> <!-- this text is on the side of the slab -->
+                    <ab>
+                        <lb n="1"/><choice><corr>ἐ</corr><sic>Η</sic></choice>ν <expan><abbr>Θ</abbr><ex>εῷ</ex></expan> <expan><abbr><am><g ref="http://sicily.classics.ox.ac.uk/alists/listglyph.xml#chi-rho">☧</g></am></abbr><ex>χριστῷ</ex></expan> Θ ΝΗ <!-- the last three letters are just decorative to create a simmetric structure with the first part? -->
```

Comment:
Maybe add the bit about the last three letters in the commentary? Allows for more explanation and potential refs to publications.

Reply:
I think that this should be reg and orig, not corr and sic on the ΗΝ ; further, the obvious and reasonable explanation of G that this is a symmetrical apotropaic text means that after the chrismon the text is repeated but reversed, i.e. written right to left rather than left to right. I'm not sure, however, how we insert the change of text-direction mid-line? with a `<milestone>` @simonastoyanova ? or a `<seg>`?

Reply:
A seg, I think.